packages/api/tests/models/granules/test-granules-model.js

```diff
@@ -282,7 +282,7 @@ test('files existing at location returns only file that exists with multiple des
   ]);
 });
 
-test('get() will translate an old-style granule file into the new schema', async (t) => {
+test('get() will translate an old-style granule file and numeric productVolume into the new schema', async (t) => {
```

Comment:
Given these are in the api model tests, we should make sure that the feature branch granules work is compatible with the updates.   The method being changed is in the api/lib directory (see comment on unit test, though), but these tests may not diretly exist in feature.